ratbag_emu/device.py

```diff
@@ -82,7 +82,7 @@ def __init__(self, name: str = 'Generic Device',
 
         self.report_rate = 100
         self.fw = Firmware(self)
-        self.hw: List[HWComponent] = []
+        self.hw: Dict[str, HWComponent] = {}
```

Comment:
Don  you need to update other parts in the code for that?

Reply:
No, this was the first commit where I started using it. I found that it was better defined as a dict than as a list, so I changed that and implemented what I needed.